protostar/scripting_runtime/script_runner.py

```diff
@@ -0,0 +1,53 @@
+import runpy
+import sys
+import traceback
+from multiprocessing import Process
+from pathlib import Path
+
+
+class ScriptRunner:
+    @staticmethod
+    def run(path: Path) -> None:
+        """
+        Runs a Python script (given by ``path``) in a subprocess forked from Protostar's process.
+        The script is **not** interpreted in sandbox mode.
+        """
```

Comment:
[Design] Will Python scripts be supported, If we rewrite Protostar in Rust?

Reply:
They should be easy to compatibly support provided that users will not import anything outside `protostar.scripting` package.